sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1227,8 +1227,18 @@ Task<Response<BlobProperties>> GetPropertiesAsync(bool async, CancellationToken
             // Download the entire stream
             async Task<Response<BlobDownloadInfo>> DownloadStreamAsync(bool async, CancellationToken ct)
             {
-                Response<BlobDownloadInfo> response = await client.DownloadAsync(accessConditions: blobAccessConditions, cancellationToken: ct).ConfigureAwait(false);
-                await response.Value.Content.CopyToAsync(destination, 81920 /* default value */, ct).ConfigureAwait(false);
+                Response<BlobDownloadInfo> response = await client.DownloadInternal(
+                    default, blobAccessConditions, default, async, cancellationToken).ConfigureAwait(false);
```

Comment:
Nit - please used a named parameter or add a `/* name */` comment if you're passing just `default` in a big argument list.